docs/resource/README.md

```diff
@@ -82,8 +82,8 @@ as specified in the [Resource SDK specification](https://github.com/open-telemet
 <!-- semconv service -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `service.name` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>Logical name of the service. [1] | `shoppingcart` | Required |
-| `service.version` | string | ![Stable](https://img.shields.io/badge/-stable-lightgreen)<br>The version string of the service API or implementation. The format is not defined by these conventions. | `2.0.0`; `a01dbef8a` | Recommended |
+| [`service.name`](../attributes-registry/service.md) | string | Logical name of the service. [1] | `shoppingcart` | Required |
```

Comment:
I'm a bit concerned that we are not rendering `stable` attribute anymore unless it's a registry

Reply:
It's a big, fixing it here https://github.com/open-telemetry/build-tools/pull/272

Reply:
hum I guess we should wait then to merge this? Given the missing stability badge. Good catch @trisch-me !

Reply:
I'm watching https://github.com/open-telemetry/build-tools/pull/272 - once it's merged, I'll rebase and regenerate the `.md` files.

Reply:
@gregkalapos PR is merged

Reply:
How can I use a version with the fix-PR of the `md` file generation?

From what I see [`make table-generation`](https://github.com/open-telemetry/semantic-conventions/blob/main/Makefile#L97) uses the [`otel/semconvgen` image](https://hub.docker.com/r/otel/semconvgen/tags). v0.23.0 definitely doesn't have the PR, tried latest but that does not generate the `![Stable]` part either.

Does anyone need to push an updated version of the docker image? Or can I do something locally myself?

Reply:
We need to wait for a release of the build tools to be published, to ultimately merge. I guess you can still do it by building it locally and changing the make target to use your local image but we won't merge it the official one is out. I think we will do that soon (hopefully)